jwql/instrument_monitors/common_monitors/readnoise_monitor.py

```diff
@@ -43,26 +43,26 @@
 from astropy.visualization import ZScaleInterval
 import crds
 import matplotlib
+import matplotlib.pyplot as plt  # noqa: E348
+import numpy as np  # noqa: E348
+from pysiaf import Siaf  # noqa: E348
+from sqlalchemy.sql.expression import and_  # noqa: E348
+
+from jwql.database.database_interface import FGSReadnoiseQueryHistory, FGSReadnoiseStats  # ruff: noqa
+from jwql.database.database_interface import MIRIReadnoiseQueryHistory, MIRIReadnoiseStats  # ruff: noqa
+from jwql.database.database_interface import NIRCamReadnoiseQueryHistory, NIRCamReadnoiseStats  # ruff: noqa
+from jwql.database.database_interface import NIRISSReadnoiseQueryHistory, NIRISSReadnoiseStats  # ruff: noqa
+from jwql.database.database_interface import NIRSpecReadnoiseQueryHistory, NIRSpecReadnoiseStats  # ruff: noqa
+from jwql.database.database_interface import session, engine
+from jwql.shared_tasks.shared_tasks import only_one, run_parallel_pipeline
+from jwql.instrument_monitors import pipeline_tools
+from jwql.utils import instrument_properties, monitor_utils
+from jwql.utils.constants import JWST_INSTRUMENT_NAMES, JWST_INSTRUMENT_NAMES_MIXEDCASE
+from jwql.utils.logging_functions import log_info, log_fail
+from jwql.utils.permissions import set_permissions
+from jwql.utils.utils import ensure_dir_exists, filesystem_path, get_config
+
 matplotlib.use('Agg')
-import matplotlib.pyplot as plt  # noqa: E348 (comparison to true)
```

Comment:
This may have been written this way because Agg needs to be set before pyplot is imported. We may need to work around this somewhere - maybe in the top level __init__

Reply:
Sounds good, I can add a noqa line before hand